src/effects/effectsmanager.h

```diff
@@ -1,146 +1,109 @@
 #pragma once
 
-#include <QObject>
 #include <QHash>
 #include <QList>
 #include <QSet>
-#include <QScopedPointer>
-#include <QPair>
 
-#include "preferences/usersettings.h"
 #include "control/controlpotmeter.h"
-#include "control/controlpushbutton.h"
+#include "effects/backends/effectsbackendmanager.h"
+#include "effects/presets/effectchainpresetmanager.h"
+#include "effects/specialeffectchains.h"
 #include "engine/channelhandle.h"
-#include "engine/effects/message.h"
+#include "preferences/usersettings.h"
 #include "util/class.h"
-#include "util/fifo.h"
 
 class EngineEffectsManager;
-class EffectChainManager;
-class EffectManifest;
-class EffectsBackend;
 
-class EffectsManager : public QObject {
-    Q_OBJECT
+/// EffectsManager initializes and shuts down the effects system. It creates and
+/// destroys a fixed set of StandardEffectChains on Mixxx startup/shutdown
+/// and creates a QuickEffectChain and EqualizerEffectChain when
+/// PlayerManager creates decks.
+class EffectsManager {
```

Comment:
I added a brief sentence noting this, but I don't want to specifically list each sub-manager class in the EffectsManager documentation because that could easily become outdated.

Reply:
Yeah, someone new would still have to read the docs for all of these sub-managers first, but I guess thats not too much to ask.